README.md

```diff
@@ -71,6 +71,29 @@ response.total     #=> Total price plus total tax
 response.subtotal  #=> Total price before tax
 ```
 
+####  Customer data
+You are required to specify a `state` or a `country`. The client will raise an error if none is specified.
```

Comment:
Is this a break change that clients upgrading will need to handle? If so, that would warrant a major version bump?

Reply:
From backward compatibility point of view - I wouldn't say so as this follows the original behaviour where clients were required to specify a `state` (`VertexClient::ValidationError` is raised otherwise). The [logic behind raising](https://github.com/customink/vertex_client/pull/51/files#diff-badc948ce7d4da5c8036cd44ca71af2d) is pretty simple, making sure we have at least one of `state` and `country`.